<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="selectFileWidget" native="true">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>879</width>
      <height>52</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>60</height>
     </size>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="selectFilePath">
       <property name="minimumSize">
        <size>
         <width>750</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>800</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectFileBtn">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="displayWi" native="true">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>1000</width>
      <height>500</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>1000</width>
      <height>500</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>500</height>
     </size>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>450</width>
         <height>450</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>450</width>
         <height>450</height>
        </size>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>155</x>
          <y>10</y>
          <width>140</width>
          <height>30</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Source Image</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="srcImgShowLabel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>211</y>
          <width>450</width>
          <height>131</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_4" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>450</width>
         <height>450</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>450</width>
         <height>450</height>
        </size>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>155</x>
          <y>10</y>
          <width>140</width>
          <height>30</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Processed</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="procImgShowLabel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>190</y>
          <width>450</width>
          <height>131</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="processTypeWidget" native="true">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>98</y>
      <width>201</width>
      <height>47</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>72</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>72</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Type:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="procTypeComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="codeWidget" native="true">
    <property name="geometry">
     <rect>
      <x>351</x>
      <y>98</y>
      <width>801</width>
      <height>52</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>60</height>
     </size>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>72</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>72</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Code:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="codeText">
       <property name="minimumSize">
        <size>
         <width>700</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>700</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="goBtn">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>110</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>GO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveBtn">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>250</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>SAVE</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
